tests/test_data.py

```diff
@@ -22,26 +22,6 @@
 aoi_bbox = (11.90, 51.46, 11.94, 51.50)
 
 queries = [
-    {
-        "datahub": Datahub.Scihub,
-        "catalog": None,
-        "platform_name": Platform.Sentinel1,
-        "date": ("20200224", "20200225"),
-        "aoi": aoi_4326,
-        "cloud_cover": None,
-        "returns_srcid": "S1A_IW_SLC__1SDV_20200224T052528_20200224T052555_031390_039CF2_BEA6",
-        "returns_uuid": "8a611d5b-f9d9-437e-9f55-eca18cf79fd4",
-    },
-    {
-        "datahub": Datahub.Scihub,
-        "catalog": None,
-        "platform_name": Platform.Sentinel1,
-        "date": ("20200502", "20200503"),
-        "aoi": aoi_wkt,
-        "cloud_cover": None,
-        "returns_srcid": "S1A_IW_GRDH_1SDV_20200502T170726_20200502T170751_032389_03BFFF_3105",
-        "returns_uuid": "a28e1042-f221-4716-8298-01bca35e6187",
-    },

```

Comment:
@MWieland I removed them because I could not find them any more. We should really implement a dummy API.